storage/gc_queue.go

```diff
@@ -239,10 +232,23 @@ func (gcq *gcQueue) process(now proto.Timestamp, repl *Replica) error {
 		}
 	}
 
+	done := true
 	if len(intents) > 0 {
+		done = false
```

Comment:
I don't think this needs to set `done` (and therefore `done` doesn't need to exist this early). If intents exist, we resolve them, and that's it.